### PR TITLE
Feat: Add code of conduct, terms and privacy policy modal

### LINKED ISSUE #176: Bug: Add links to Code of Conduct, Terms & Privacy Policy in the Register UI
**Describe the bug**
User can't go through the AnitaB.org's Code of Conduct, Terms & Privacy Policy during registering. 

**To Reproduce**
1. Go to http://localhost:3000/register
2. See error

**Expected behavior**
Add links to AnitaB.org's Code of Conduct, Terms & Privacy Policy in the Register UI or add a Modal to display  Code of Conduct, Terms & Privacy Policy.

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Any
 - Browser: Any
 - Version: Any

### PR DESCRIPTION
**Pull request by Rahulm2310**
### Description

Add Code of Conduct, Terms and Privacy Policy modal popup on the register page.

Fixes #176 

### Type of Change:

- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

bit-terms-modal-2

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Rahulm2310** [comment]
@mtreacy002 Please review this PR. Thanks 🙂

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@Rahulm2310 , can you please resolve the conflict first? I'll give my review once you've done this. Thanks

**Comment by Rahulm2310** [comment]
@mtreacy002 removed the conflicts 🙂. Please review it now

**Comment by mtreacy002** [comment]
> @mtreacy002 Done 🙂. Please review it now

will do that today 😉.

**Review by mtreacy002 — COMMENTED** [review]
@Rahulm2310 , I think it looks good now. Thank you for the fix, I just need to confirm with @anitab-org/admins that this code of conduct, terms and privacy policy can be used in BridgeInTech project. Also, can you please provide the links to the pages where you find the contents in your reply to this comment? Thanks beforehand.

**Comment by Rahulm2310** [comment]
@mtreacy002 Sure. Here are the links to the pages 
1. [https://anitab.org/privacy-policy/](https://anitab.org/privacy-policy/)
2. [https://anitab.org/terms-of-use/](https://anitab.org/terms-of-use/)
3. [https://ghc.anitab.org/code-of-conduct/](https://ghc.anitab.org/code-of-conduct/)

Also, I think Mentorship System is also using the same links for the project.

**Comment by mtreacy002** [comment]
> @mtreacy002 Sure. Here are the links to the pages
> 
> 1. https://anitab.org/privacy-policy/
> 2. https://anitab.org/terms-of-use/
> 3. https://ghc.anitab.org/code-of-conduct/
> 
> Also, I think Mentorship System is also using the same links for the project.

thanks for the links, @Rahulm2310 . In tonight's open session we could ask confirmation from @isabelcosta who will hopefully join us there 😉.

File: src/register/TermsAndPrivacyPolicyModal.jsx
Comment (mtreacy002) [inline src/register/TermsAndPrivacyPolicyModal.jsx:11]: @Rahulm2310 , please replace the content for the Code of Conduct to [this one](https://github.com/anitab-org/documentation/blob/master/Contributing/CODE_OF_CONDUCT.md) from the Code of Conduct under anitab-org/documentation as @isabelcosta referred to at tonight's Open Session. The one you provide here is specific to GHC code of conduct only.

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@Rahulm2310 , please check my feedback below.

**Comment by Rahulm2310** [comment]
@mtreacy002 Applied suggested changes. Please have a look now. 🙂

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@Rahulm2310 , thank you for the fix. Can you please squash your commits into one? I will write the test report and approve after that.

**Comment by Rahulm2310** [comment]
@mtreacy002 Is it okay now ?

**Comment by mtreacy002** [comment]
> @mtreacy002 Is it okay now ?

@Rahulm2310 , how come it's still showing 2 commits 😁?

**Comment by Rahulm2310** [comment]
@mtreacy002 Actually, I squashed my commits and pushed my changes but there were conflicts which I then resolved using GitHub GUI.

**Comment by mtreacy002** [comment]
@Rahulm2310 , it is good to develop the habit to "pull --rebase" from upstream and resolve any conflicts before you squash your commits and push the changes again to avoid being in this situation. But that's ok, with AnitaB.Org Open Source, maintainer actually can squash the commits at our end 😁. I just think  it's good to learn the efficient workflow as other Open Source organizations might have different GitHub settings/rules and expects contributors to squash the commits from their end 😁. 
Would you like to me to squash the commits at my end for now or do you want to squash the commits again at your end?

**Comment by Rahulm2310** [comment]
@mtreacy002 learn a lot here 😊. Thanks, will try it at my end 👍

**Comment by Rahulm2310** [comment]
@mtreacy002 Please review it now. 🙂

**Review by codesankalp — COMMENTED** [review]
@Rahulm2310 @mtreacy002 
I have a suggestion.
Instead of adding all the terms and privacy policy in the form of jsx in `TermsAndPrivacyPolicyModal.jsx` why don't we use a markdown parser to parse these markdowns from their actual source when loading the modal.
If we will use this then whenever their is a change in code of conduct, terms, and privacy policy then it will automatically change on the webpage instead of manually changing it again and again.

**Comment by mtreacy002** [comment]
That sounds like a great idea, @codesankalp. @Rahulm2310 will you be able to accommodate this?

**Comment by Rahulm2310** [comment]
@codesankalp seems a nice idea but what do you mean by using a markdown parser. Did you mean scraping ?

**Comment by codesankalp** [comment]
@Rahulm2310 Just fetch the content from GitHub Readme or markdown files.
Example:
Like fetch the content from this README URL (https://raw.githubusercontent.com/anitab-org/bridge-in-tech-web/develop/README.md) and then convert it into HTML by using markdown parser (https://www.npmjs.com/package/markdown-it).
We can't say it scraping because scraping is different, this is just fetching and parsing.

**Comment by Rahulm2310** [comment]
@codesankalp Only the [Code of Conduct](https://github.com/anitab-org/documentation/blob/master/Contributing/CODE_OF_CONDUCT.md) is a markdown file. [Terms](https://anitab.org/terms-of-use/) & [Privacy Policy](https://anitab.org/privacy-policy/) are present on AnitaB.org's official website.

**Comment by mtreacy002** [comment]
@codesankalp, what @Rahulm2310 says make sense. I've [asked admin](https://anitab-org.zulipchat.com/#narrow/stream/216326-documentation/topic/anitab-org.2Fdocumentation/near/228503958) if we can open an issue to create .md files for these Terms of Use and Privacy Policy. We can refactor this code later once that issue is solved. I'll approve this issue now

**Review by mtreacy002 — APPROVED** [review]
Well done, @Rahulm2310 for adding this feature. Below is the test result:
1. Code review: done
2. All possible responses were tested as below:
    * Test PR 179 - add code of conduct, terms of use and privacy policy modal
    Expected behaviour: When user click on the link to Code of Conduct, Terms and Privacy Policy, a modal pop-up should open with content of the 3 policies above.
    Actual behaviour: As expected

ezgif com-gif-maker (34)

3. Environment:
    OS: macOS Catalina v10.15.7
    Browser: Chrome v88.0.4324.192

Thank you again, @Rahulm2310 for contributing to the project 🎉🎉🎉

**Pull request merged**
